test/parallel/test-http-early-hints.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const common = require('../common');                                   // 2
```

Comment:
```suggestion
const common = require('../common');
```

Reply:
😅 how did that get in there, thanks for noticing